lib/install/management/delete.go

```diff
@@ -143,6 +150,46 @@ func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *metadata.V
 	return nil
 }
 
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) error {
+	if d.force {
+
+		for label, url := range conf.VolumeLocations {
+
+			// separate the host from the path in the provided path URL
+			pathAry := strings.SplitN(url.Path, " ", 2)
+			if len(pathAry) != 2 {
+				return errors.New("Didn't receive an expected volume store path format")
+			}
+
+			// convert the URL path to vSphere style pathing, and omit everything from the path except the root of the volume store
+			vSpherePath := fmt.Sprintf("%s %s", pathAry[0], strings.Split(pathAry[1], "/")[0])
+
+			// connect to vSphere and do the actual deletion
+			m := object.NewFileManager(d.session.Vim25())
```

Comment:
There is method in store_files, deleteDatastoreFiles, which will handle file not found scenario, and return correct error message.
Please change to call that method here.